### ISSUE TITLE
[Bug] wxGUI: profile not shown with Python 3.10

### ISSUE THREAD

**Issue by neteler** [OSGeo/grass#2043]
**Describe the bug**

When attempting to draw an elevation model profile, the plot lacks the actual profile:

![image](https://user-images.githubusercontent.com/1295172/147533289-728a04de-399d-43fe-b498-cf611ead9153.png)

The console shows this warning (not sure if related):

```
/usr/lib64/python3.10/site-
packages/wx/lib/plot/utils.py:257: PlotPendingDeprecation:
`SetShowScrollbars` is pending deprecation. Please use
`self.showScrollbars property` instead.
  _warn(warn_txt.format(inspect.stack()[1][3], new_func),
```

**To Reproduce**

1. Load NC sample dataset, some elevation map
2. Start GRASS Profile Analysis Tool
3. draw line

**System description (please complete the following information):**

- Operating System: Fedora 35, wxPython: 4.0.7 with 4.1 patches
- GRASS GIS version: both 7.8.dev and 8.0.dev

System Info                                                                     
```
GRASS version: 8.0.dev                                                          
Code revision: ff485ffdc                                                        
Build date: 2021-12-14                                                          
Build platform: x86_64-pc-linux-gnu                                             
GDAL: 3.3.2                                                                     
PROJ: 8.2.0                                                                     
GEOS: 3.9.2                                                                     
SQLite: 3.36.0                                                                  
Python: 3.10.1                                                                  
wxPython: 4.0.7                                                                 
Platform: Linux-5.15.6-200.fc35.x86_64-x86_64-with-glibc2.34
```

**Comment by tmszi** [comment]
This warning message is Deprecation warning message. I observed the same behavior "nothing is drawn" at the wxGUI Cartographic Composer component (after applying this PR #2050), when you add a vector/raster map to Draft tab mode, nothing is drawn (without error message), but when you render the result on the Preview tab, the map is rendered.

**Comment by petrasovaa** [comment]
This is bug in wx, I created a PR https://github.com/wxWidgets/Phoenix/pull/2065, @swt2c could you look at it? 
I didn't address the warning.
I think this could be closed, nothing to do in GRASS here.

**Comment by petrasovaa** [comment]
The wx PR was merged, closing.

**Comment by neteler** [comment]
Excellent, your PR https://github.com/wxWidgets/Phoenix/pull/2065 solves it here (I patched my python3-wxpython4-4.0.7-24.fc35 installation). Hopefully @swt2c will backport it.

Thanks @petrasovaa!

![image](https://user-images.githubusercontent.com/1295172/148290748-6375a7e0-dc79-4f9d-8248-0416ce79820a.png)

**Comment by swt2c** [comment]
@neteler will do.  I found a few other Py3.10 issues, want to include those as well.
